cactusbot/handlers/events.py

```diff
@@ -102,29 +106,74 @@ def __init__(self, cache_data, api):
     async def on_host(self, packet):
         """Handle host packets."""
 
-        if self.alert_messages["host"]["announce"]:
-            return MessagePacket(
-                self.alert_messages["host"]["message"].replace(
-                    "%USER%", packet.user
-                ))
+        if not self.alert_messages["host"]["announce"]:
+            return
+
+        response = MessagePacket(
+            self.alert_messages["host"]["message"].replace(
+                "%USER%", packet.user
+            ))
+
+        if packet.success:
```

Comment:
Oh, and considering we have a bunch of cached things now, it's probably better to just use `self.cache_config = cache_data`, rather than a wall of `self.cache_follows = cache_data["CACHE_FOLLOWS"]`.